recipe/meta.yaml

```diff
@@ -11,12 +11,13 @@ source:
   sha256: f0e32cb10654fea91ad56bde89170d78cfbf4363ee0b01d8f097de2ba49f6ce9
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
-    - cmake
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
```

Comment:
cmake and ninja need to move to `host:`

Reply:
Apologies, I didn't read the build failure closely enough.

The failure is in the test - you probably need to set:
`test: native_and_emulated` in the `conda-forge.yml`

You also probably need to move cmake/ninja back where it was.

Reply:
No worries. I'm new to this tool, so I appreciate any guidance. Hopefully, CI will now turn green!